change/@azure-communication-react-0f1323bf-fac2-4405-afa6-1dafb593ba43.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
```

Comment:
It looks like the stable change log item is lost

Reply:
What do you mean by this?

Reply:
There always should be 2 changelog files, 1 for beta releases, 1 for stable. The beta (not stable as mentiones in the previous comment) changelog item is not commited in this PR